src/pages/components/dropdown/usage.mdx

```diff
@@ -641,7 +671,37 @@ Use when you can select multiple options from a list or to filter information.
 </Column>
 </Row>
 
-#### Clear all
+#### Selecting all
+
+The parent checkbox associated to the parent option makes it possible for user
+to select all options from the list. Although the parent checkbox element is
+used only for multiselect dropdowns, it is not mandated to be present in all
+multiselect dropdowns.
+
+- The bottom stroke for the parent option has no padding and bleeds to the end
+  of the dropdown menu.
+- The background fill for the all button does not change when selected and will
+  continue to have the same background as the active state
```

Comment:
```suggestion
  continue to have the same background as the enabled state.
```

Reply:
changed word to enabled to maintain consistency with how the component is documented